iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -106,6 +113,26 @@ public override async Task OpenAsync(CancellationToken cancellationToken)
             }
         }
 
+        private void checkTimeout(Object _)
+        {
+            if (_twinResponseTimeouts.Any())
+            {
+                var currentDateTime = DateTime.UtcNow;
+                TimeSpan difference;
+                foreach (KeyValuePair<string, DateTime> entry in _twinResponseTimeouts)
+                {
+                    difference = currentDateTime - entry.Value;
+                    if (difference >= _twinResponseTimeout)
+                    {
+                        if (_twinResponseCompletions.ContainsKey(entry.Key))
+                            _twinResponseCompletions.TryRemove(entry.Key, out TaskCompletionSource<Twin> _);
```

Comment:
Except for logging, we always use braces.
```suggestion
                        if (_twinResponseCompletions.ContainsKey(entry.Key))
                        {
                            _twinResponseCompletions.TryRemove(entry.Key, out TaskCompletionSource<Twin> _);
                        }
```

Reply:
Irrelevant if you remove the ConstainsKey check.